### ISSUE TITLE
Error: Invalid header magic, is the host running?

### ISSUE THREAD

**Issue by LaLadaLa** [gnif/LookingGlass#66]
### Required information

Host CPU: Intel Core i5-6500
Host GPU: iGPU
Guest GPU: ASUS STRIX-GTX950-DC2OC-2GD5-GAMING, 2GB
Host Kernel version: Manjaro (base Archlinux), Kernel 4.16.7-1
Host QEMU version: 2.12.0-1

### Description of problem:
The looking-glass-client application ending immediately after starting.

### Steps to Reproduce:
1.) Start guest "qemu-system-x86_64 -machine pc-q35-2.11,accel=kvm ... -drive if=pflash,format=raw,readonly,file=./OVMF_CODE-pure-efi.fd ... -global ICH9-LPC.disable_s4=0 ..."
2.) looking-glass-client -p 5902 -F (It works well.)
3.) Hibernate guest (S4)
4.) Reset host
5.) Start guest
6.) looking-glass-client -p 5902 -F
The following information is displayed:
```
[I]               main.c:697  | run                            | Looking Glass (a10-35-g3d9d275d61)
[I]               main.c:698  | run                            | Locking Method: Atomic
[I]               main.c:691  | try_renderer                   | Using Renderer: OpenGL
[I]               main.c:779  | run                            | Using: OpenGL
[I]               main.c:897  | run                            | Waiting for host to signal it's ready...
[I]              spice.c:341  | spice_on_common_read           | notify message: keyboard channel is insecure
[I]               main.c:901  | run                            | Host ready, starting session
[E]               main.c:906  | run                            | Invalid header magic, is the host running?
```

### Actual results:
The looking-glass-client app closes.

I have to reset Looking-glass-host.exe.

**Comment by cmdrkotori** [comment]
I took the liberty of doing the research for you. :) I believe you check for [PBT_APMRESUMEAUTOMATIC](https://docs.microsoft.com/en-us/windows/desktop/power/pbt-apmresumeautomatic). A callback for a thread can be registered with [PowerRegisterSuspendResumeNotification](https://docs.microsoft.com/en-us/windows/desktop/api/Powerbase/nf-powerbase-powerregistersuspendresumenotification).
Cheers.

**Comment by ipatix** [comment]
I'm not sure if my problem is related, but I also get the same error in the log.

Basically, I start up my VM with the looking glass host running automatically on startup, I try to connect from my VM host with the looking glass client, and I get the error above.

So, looking glass doesn't appear to work for me at all. What does it have to do with host resetting?

One more thing I can observe is that whenever I initially launch the looking glass client, the QXL display in virt-manager turns off (which is currently used as secondary display for an Nvidia GPU).

**Comment by gnif** [comment]
@ipatix seek help via the L1tech forums or the LG Discord as per the instructions. This issue is well understood and is not related to yours.

**Comment by gnif** [comment]
This issue should now be fixed as of 6f39434. Closing but will re-open if it's reported again.
